src/app/content-area/content-area.component.ts

```diff
@@ -386,36 +394,56 @@ export class ContentAreaComponent implements AfterContentInit {
     @HostListener('window:resize', ['$event'])
     @HostListener('window:load', ['$event'])
     private onWindowResize(evt: Event): void {
-        let headerNode: Element = document.getElementsByClassName('header navbar')[0];
+        let messagePane: HTMLElement = document.getElementById('message-pane-div');
         let contentArea: Element = document.getElementsByClassName('row content')[0];
         let isReadingPane: boolean = (contentArea.className.indexOf('show-reading-pane') === -1);
         if (!isReadingPane && window.innerWidth < 605) {
             return;
         }
         if (window.innerWidth < 1200) {
-            headerNode.className = 'header navbar head-pane-hide';
             let headerRP: Element = document.getElementsByClassName('header-right-pane selected')[0];
             if (headerRP) {
                 headerRP.className = 'header-right-pane';
             }
             contentArea.className = 'row content';
+            this.popupSideBar.type="Over";
         } else {
-            headerNode.className = 'header navbar';
             if (contentArea.className.indexOf('show-header-content') === -1) {
                 contentArea.className = 'row content';
             } else {
                 contentArea.className = 'row content show-header-content';
             }
+            this.popupSideBar.type="Push";
         }
         if (window.innerWidth < 1090) {
             contentArea.className = 'row content sidebar-hide';
+            messagePane.classList.remove("msg-top-margin");
+            this.sidebarSection.type = "Over";
+            this.sidebarSection.sidebar.showBackdrop=true;
         } else {
+            messagePane.classList[ this.data.isNewMailClick ? 'add' : 'remove' ]('msg-top-margin');
             this.hideSideBar();
+            this.sidebarSection.type = "Push";
+            this.sidebarSection.sidebar.showBackdrop=false;
+            this.sidebarSection.showSidebar();
         }
         if (window.innerWidth < 605) {
             if (isReadingPane) {
                 contentArea.className = contentArea.className + ' ' + 'show-message-pane';
             }
+            messagePane.classList.remove("msg-top-margin");
+            if (!messagePane.classList.contains('msg-top-margin')) {
+                this.data.showEmptyMessage();
+            }
+            this.defaultSplitter.isDesktopMode=false;
+            this.bindEvent();
```

Comment:
Check and avoid this method here

Reply:
Event Emitter is used to Avoid the method.